source/_integrations/smartenergy_goecharger.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: go-e Charger Cloud
+description: Instructions on setting up go-e Charger within Home Assistant.
+ha_category:
+  - Car
+  - Sensor
+  - Button
+ha_iot_class: Cloud Polling
+ha_quality_scale: silver
+ha_release: '2023.2'
+ha_config_flow: true
+ha_codeowners:
+  - '@openkfw'
+ha_domain: smartenergy_goecharger
+ha_platforms:
+  - button
+  - number
+  - select
+  - sensor
```

Comment:
The addition of new integrations must be limited to a single platform.
More platforms can be added later in followup PRs.

Reply:
Is it possible to add more platforms after the initial PR or even after that one platform per one PR?